src/uct/ib/rc/base/rc_ep.h

```diff
@@ -32,6 +32,12 @@ enum {
     UCT_RC_TXQP_STAT_LAST
 };
 
+enum {
+    UCT_RC_EP_ADDR_TYPE_BASIC,
+    UCT_RC_EP_ADDR_TYPE_TM,
```

Comment:
what TM stand for ? Since the naming is not self descriptive I would suggest to add few comments.